docker-compose.yml

```diff
@@ -81,6 +81,8 @@ services:
       - LOG_LEVEL=trace
       - DB_HOST=postgres-veritable-ui-bob
       - DB_NAME=veritable-ui
+      - API_SWAGGER_TITLE=Bob
+      - API_SWAGGER_BG_COLOR=#ff3131
```

Comment:
As per story I was no able to find `LABEL`, only in cloudagent. Used API_TITLE and API_BG_COLOR instead